### ISSUE TITLE
QDesktopWidget::availableGeometry is deprecated

### ISSUE THREAD

**Issue by tastytea** [Nheko-Reborn/nheko#107]
### System:

- Nheko version: 0.6.4 and “master”-branch, commit 1de7efd4eaaaf0b9ac9da1e19bbbc8acad0133af
- Installation method: Local build (I also tried the Gentoo ebuild from the “matrix” overlay)
- Operating System: Gentoo Linux
- Qt version: 5.13.2
- C++ compiler: g++ (Gentoo 9.2.0-r2 p3) 9.2.0

### Actual behavior

Local build failed with:

```
[  1%] Building CXX object CMakeFiles/nheko.dir/src/dialogs/ImageOverlay.cpp.o
/var/ram/nheko/src/dialogs/ImageOverlay.cpp: In constructor 'dialogs::ImageOverlay::ImageOverlay(QPixmap, QWidget*)':
/var/ram/nheko/src/dialogs/ImageOverlay.cpp:42:62: error: 'const QRect QDesktopWidget::availableGeometry(int) const' is deprecated: Use QGuiApplication::screens() [-Werror=deprecated-declarations]
   42 |         screen_ = QApplication::desktop()->availableGeometry();
      |                                                              ^
In file included from /usr/include/qt5/QtWidgets/QDesktopWidget:1,
                 from /var/ram/nheko/src/dialogs/ImageOverlay.cpp:19:
/usr/include/qt5/QtWidgets/qdesktopwidget.h:88:67: note: declared here
   88 |     QT_DEPRECATED_X("Use QGuiApplication::screens()") const QRect availableGeometry(int screen = -1) const;
      |                                                                   ^~~~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
gmake[2]: *** [CMakeFiles/nheko.dir/build.make:485: CMakeFiles/nheko.dir/src/dialogs/ImageOverlay.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:73: CMakeFiles/nheko.dir/all] Error 2
gmake: *** [Makefile:130: all] Error 2
```

### Steps to reproduce

- Build Nheko with Qt 5.13.2

### Logs

[build.log](https://github.com/Nheko-Reborn/nheko/files/4059452/build.log) (From the Gentoo ebuild)

**Comment by deepbluev7** [comment]
That should be fixed in the 0.7.0-dev branch already. If you don't want to use that, you should be able to add -Wno-error to the cflags, I think.

**Comment by tastytea** [comment]
haven't tried 0.7.0-dev but without `-Werror` it compiles.
